site/e2e/global.setup.ts

```diff
@@ -4,7 +4,7 @@ import { STORAGE_STATE } from "./playwright.config"
 import { Language } from "../src/components/CreateUserForm/CreateUserForm"
 
 test("create first user", async ({ page }) => {
-  await page.goto("/", { waitUntil: "networkidle" })
+  await page.goto("/", { waitUntil: "domcontentloaded" })
```

Comment:
I don't think we should necessarily change `networkidle` to `domcontentloaded` without adding additional wait checks. For instance "wait until element X is present in the html". In a sense, `networkidle` should be better as it will wait longer compared to `domcontentloaded` which will fire immediately after dom even if network requests are still outbound.

Reply:
In previous PRs, I added extra "wait until" conditions, so in theory it should be faster and still safe with `domcontentloaded`.

Reply:
Oh, that's perfect 👍🏻